P5/Test/Makefile

```diff
@@ -107,19 +107,29 @@ detest: checkrnv
 	@echo " "
 	@echo "--------- work on target $@ ---------"
 	@echo BUILD: Test file with deliberate mistakes
+	@echo "--- run ant to validate ODD file (./detest.odd):"
+#	Note that output below is called 'detest_odd_schematron' because if there
+#	is a RelaxNG error the build dies and no output is generated
+	$(ANT) -f antruntest.xml -Doutputname=detest -DoddFile=detest.odd validateodd > detest_odd_schematron.log 2>&1
+	@echo "--- run ant to compile ODD file inoto RelaxNG and Schematron"
+	@echo "(detest.odd → detest.rng & detst.isosch):"
```

Comment:
Shouldn't that be "detest.isosch"?

Reply:
Yup.

Reply:
Still working on other bugs, will fix that typo, too, in a few mins.